File: Ambulance.BLL/Models/DecAllergAnalyticsModel.cs
namespace Ambulance.BLL.Models;

public class DecAllergAnalyticsModel
{
    // назва захворювання та кількість пацієнтів із цим захворюванням
    public Dictionary<string, int> DeceaseStatistics { get; set; } = new();

    // назва алергії та кількість пацієнтів із цією алергією
    public Dictionary<string, int> AllergyStatistics { get; set; } = new();
}

File: src/api/models/AnaliticsModels/DecAllergAnalyticsModel.js
// models/decAllergAnalyticsModel.js
export class DecAllergAnalyticsModel {
    constructor({ deceaseStatistics = {}, allergyStatistics = {} } = {}) {
        // Об'єкти у форматі {назва: кількість}
        this.deceaseStatistics = deceaseStatistics;
        this.allergyStatistics = allergyStatistics;
    }
}
